test/contrib/bigquery_gcloud_test.py

```diff
@@ -27,23 +27,30 @@
 
 import luigi
 import unittest
-from luigi.contrib import bigquery, gcs
 
 try:
     import googleapiclient.errors
     import oauth2client
 except ImportError:
     raise unittest.SkipTest('Unable to load googleapiclient module')
+from luigi.contrib import bigquery, bigquery_avro, gcs
+import avro.schema
+from avro.datafile import DataFileWriter
+from avro.io import DatumWriter
 
+from contrib import gcs_test
 from nose.plugins.attrib import attr
+from helpers import unittest
 
 # In order to run this test, you should set your GCS/BigQuery project/bucket.
 # Unfortunately there's no mock
-PROJECT_ID = os.environ.get('BQ_TEST_PROJECT_ID', 'your_project_id_here')
-BUCKET_NAME = os.environ.get('BQ_TEST_INPUT_BUCKET', 'your_test_bucket_here')
+PROJECT_ID = gcs_test.PROJECT_ID
```

Comment:
Tests were failing after the merge. End reason was because tests inpullrequest https://github.com/spotify/luigi/pull/1917/ did not clean up. And then did not revert this part of debugging.